ignite-base/App/Services/RehydrationServices.js

```diff
@@ -18,14 +19,14 @@ const updateReducers = (store) => {
         preview: 'Reducer Version Change Detected',
         important: true
       })
-      // Purge store
-      persistStore(store, config).purge()
+      // Purge store and dispatch startup
+       persistStore(store, config,()=> {store.dispatch(StartupActions.startup());}).purge()
       AsyncStorage.setItem('reducerVersion', reducerVersion)
     } else {
-      persistStore(store, config)
+      persistStore(store, config,()=> {store.dispatch(StartupActions.startup());});
```

Comment:
Please remove semi-colons.  We're more of a whitespace vs semicolons kinda paradigm throughout the app, to remain Standard JS complaint.